frontend/src/components/Post.js

```diff
@@ -13,15 +13,51 @@ const Post = props => {
     props.onComment(props.post._id, commentData)
   }
 
+  const originalVotes = props.post.upVotes - props.post.downVotes;
+  const [votes, toggle] = React.useState(originalVotes);
+
+
+  const clickUpVote = () => {
+    if (votes>originalVotes) {
+      toggle(originalVotes);
+      props.updateUpVote(false);
+    } else {
+      if (votes<originalVotes) {
+        props.updateDownVote(false);
+      }
+      toggle(originalVotes+1);
+      props.updateUpVote(true);
+    }
+  }
+  const clickDownVote = () => {
+    if (votes<originalVotes) {
+      toggle(originalVotes);
+      props.updateDownVote(false);
+    }
+    else { 
+      if (votes>originalVotes) {
+        props.updateUpVote(false);
+      }
+      toggle(originalVotes-1);
+      props.updateDownVote(true);
+    }
+  }
+
+  const updateVoteColor = () => {
+    if (votes>originalVotes) return 'orange';
```

Comment:
I'd refactor this to always apply the `center` class, so you don't have to write `text-align: center` for all three styles. So You could say `return 'orange center'` and `return 'blue center'` in this case!